modules/jlink/artifacts/opt/jboss/container/java/jlink/mkdeps.sh

```diff
@@ -3,6 +3,7 @@ set -euo pipefail
 shopt -s globstar
 
 function generate_deps() {
+  exit 2
```

Comment:
I guess this is a stray line from testing?

Ideally we can write some behave unit tests. We can put known-to-fail quickstarts for testing in https://github.com/rh-openjdk/openjdk-container-test-applications . I won't block this PR on adding unit tests, we're tracking what coverage to have in another JIRA.